codec/encoder/core/inc/param_svc.h

```diff
@@ -426,6 +432,11 @@ typedef struct TagWelsSvcCodingParam: SEncParamExt {
       pSpatialLayer->uiTransferCharacteristics = pCodingParam.sSpatialLayers[iIdxSpatial].uiTransferCharacteristics;
       pSpatialLayer->uiColorMatrix =             pCodingParam.sSpatialLayers[iIdxSpatial].uiColorMatrix;
 
+      pSpatialLayer->bAspectRatioPresent = pCodingParam.sSpatialLayers[iIdxSpatial].bAspectRatioPresent;
+      pSpatialLayer->eAspectRatio = pCodingParam.sSpatialLayers[iIdxSpatial].eAspectRatio;
+      pSpatialLayer->sAspectRatioExtWidth = BIG_ENDIAN_16(pCodingParam.sSpatialLayers[iIdxSpatial].sAspectRatioExtWidth);
```

Comment:
1. should remove BIG_ENDIAN_16 macro. Currently openh264 default uses little endian mode.
2. if support BIG endian, this should be put writebits function. right?